### DISCUSSION TITLE
how to create a 'nothing' transition

### DISCUSSION THREAD

**Discussion by justme-1968** [davdroman/swiftui-navigation-transitions#124] [Q&A] [answered]
i was wondering if there is a way to create a (custom) transition that does nothing. i.e. immediately jumps from one view to the next.

for transitioning between two views this appearance can be achieved by clearing and immediately setting the navigation path. 

but for the transition from an empty path to a single element or from a single element to to an empty path the default transition is always used.

thanks
  andre

**Comment by davdroman** [comment, marked as answer]
`.navigationTransition(.default.animation(.none))`

**Reply by justme-1968** [reply]
thank you! works like a charm.

and something like:
```
extension AnyNavigationTransition {
  static var none: Self {
    .default.animation(.none)
  }
}
```
makes it look nice.
